tuf-spec.md

```diff
@@ -1505,10 +1505,10 @@ it in the next step.
    metadata file found earlier in step [[#update-targets]].  In either
    case, the client MUST write the file to non-volatile storage as FILENAME.EXT.
 
-# 6. Usage # {#usage}
+# Repository operations # {#repository-operations}
```

Comment:
Not directly related to this PR, but would you mind changing:
- `section 6.1` to `Section [[#key-management-and-migration]]` in [tuf-spec.md#L307](https://github.com/theupdateframework/specification/blame/6fc41b93a4a31914348b8b2baaa5d2cf126c5644/tuf-spec.md#L307), and 
- `Section 7` to `Section [[#consistent-snapshots]]` in [tuf-spec.md#L692](https://github.com/theupdateframework/specification/blame/6fc41b93a4a31914348b8b2baaa5d2cf126c5644/tuf-spec.md#L692)
- ?

Reply:
Nice. Done.